state-chain/chains/src/dot.rs

```diff
@@ -298,6 +298,7 @@ impl FeeEstimationApi<Polkadot> for PolkadotTrackedData {
 	fn estimate_ingress_fee(
 		&self,
 		_asset: <Polkadot as Chain>::ChainAsset,
+		_fee_multiplier: FixedU128,
```

Comment:
If we add this we should use it and then set the multiplier to 1 in the config. Otherwise changing it has not effect.